source/backtraceClient.ts

```diff
@@ -37,11 +38,17 @@ export class BacktraceClient {
   public memorize(key: string, value: any): void {
     (this.options.userAttributes as any)[key] = value;
   }
-
-  public createReport(payload: Error | string, reportAttributes: object | undefined = {}): BacktraceReport {
+  
+  public createReport(
+      payload: Error | string,
+      reportAttributes: object | undefined = {}, 
+      attachment?: string | object,
```

Comment:
How our users can use attachments in backtrace-js? 

For electron, they need to enable nodejs intgration probably if someone will pass a path to file, but is this possible in web browser?

Reply:
I'll update README to explain how to use attachments now.